tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -72,7 +72,10 @@ def test_memory_exhaustion(self, duthosts, enum_rand_one_per_hwsku_hostname, loc
         # Verify DUT triggered OOM reboot.
         self.wait_until_reboot(duthost, datetime_before_reboot)
         # Wait until all critical processes are healthy.
-        wait_critical_processes(duthost)
+        timeout = 300
+        if len(duthost.frontend_asics) > 1:
```

Comment:
Please use is_multi_asic instead of asserting frontend_asics, so that we can cover the devices with backend_asics also